### DISCUSSION TITLE
one user has a 500 error

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [nebari-dev/nebari#217] [Q&A] [answered]
We currently have one user who gets a 500 error after entering her github password. 
She's using a "incognito tab", and when she tried logging in, I see nothing in the logs viewed with k9s.
I don't see anything wrong with the cluster.

Any idea of what to do?

**Comment by dharhas** [comment]
Folks are pretty much out this week and many are next week as well. @tylerpotts can you respond to this when you are able. Pretty sure you've dealt with diagnosing this type of problem the most.

@rsignell-usgs anything unusual about her username, i.e. hyphens etc.

**Comment by rsignell-usgs** [comment]
@dharhas, nothing special about the username for this user, but I did have her with primary group admin, then switched her primary group in the config.   There is no problem with that is there? 

I'm thinking I will try just deleting her from the config and then adding her again.   Does that seem like the best solution for now?

**Comment by dharhas** [comment]
@rsignell-usgs did deleting and adding her work?

**Comment by rsignell-usgs** [comment, marked as answer]
@dharhas When I went to remove her, I noticed there was a typo in the config, "ESIPLab" instead of "ESIPlab" (grrr...) so I committed the change to the config, but the github action failed due to the [helm chart issue](https://github.com/Quansight/qhub/issues/225), so I haven't been able to test it!    

I'm not sure what to do.  Waiting for advice here: https://github.com/Quansight/qhub/discussions/218#discussioncomment-262744

**Reply by tylerpotts** [reply]
This was resolved in https://github.com/Quansight/qhub/issues/225#issuecomment-756134292

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
